pkg/controllers/netconf/network_config_controller.go

```diff
@@ -63,9 +62,11 @@ func (n *NetworkConfigController) Run(stopCh <-chan struct{}, wg *sync.WaitGroup
 }
 
 func (n *NetworkConfigController) ensure() {
-	for _, c := range n.configs {
-		if err := c.Ensure(); err != nil {
-			glog.Errorf("found an error: %v when ensuring %v", err, reflect.ValueOf(c))
+	for _, cs := range n.configSets {
+		for _, c := range cs.Configs {
+			if err := c.Ensure(cs.Enabled); err != nil {
+				glog.Errorf("found an error: %v when ensuring %v", err, reflect.ValueOf(c))
```

Comment:
nit: log cs.FeatureName as well

also considering add a vlog for feature debugging

Reply:
Logged the cs.FeatureName.
I will have other PRs for loggings.